src/main/webapp/app/entities/programming-exercise/programming-exercise.component.ts

```diff
@@ -106,4 +117,16 @@ export class ProgrammingExerciseComponent extends ExerciseComponent implements O
             reason => {},
         );
     }
+
+    editIntIntelliJ(programmingExercise: ProgrammingExercise) {
+        this.javaBridge.editExercise(stringifyCricular(programmingExercise));
+    }
+
+    openIntelliJEditor(exericse: ProgrammingExercise) {
+        try {
+            this.router.navigate(['code-editor', 'ide', exericse.id, 'admin', 'test']);
```

Comment:
make it consistent with the experience of edit in editor in Artemis

Reply:
The default route now also points to the template participation